ApplicationServerDeployment/lambda/alexa/discovery.py

```diff
@@ -0,0 +1,86 @@
+import os
+
+import boto3
+import requests
+
+from response import AlexaResponse
+import logging
+import config
+import utils
+logger = logging.getLogger(__name__)
+
+
+class DiscoveryControl(object):
+
+	def __init__(self, event=None, namespace=None):
+		if event is not None:
+			self._auth_token = event['directive']['payload']['scope']['token']
+			self._event = event
+			self._namespace = namespace
+
+	def get_discovery_response(self):

```

Comment:
add block comment to explain the purpose of the function

Reply:
done